src/containers/notification/components/desktop/UserHeader.tsx

```diff
@@ -38,19 +40,34 @@ export const UserImage = ({
   onProfileClick: (handle: string) => void
   className?: string
 }) => {
+  const [loadImage, setLoadImage] = useState(false)
   const profilePicture = useUserProfilePicture(
     user.user_id,
     user._profile_picture_sizes,
-    SquareSizes.SIZE_150_BY_150
+    SquareSizes.SIZE_150_BY_150,
+    undefined,
+    undefined,
+    loadImage
   )
 
+  // Loading the images immediately causes lag in the NotificationPanel animation
+  useEffect(() => {
+    if (!loadImage) {
+      const t = setTimeout(() => {
```

Comment:
Considered hooking into the Popup animation end to trigger this. But that would require changes to the Popup so I opted to keep it simple with a timeout for now